### ISSUE TITLE
Get contact visits by date?

### ISSUE THREAD

**Issue by CraigH78** [tendio/API#1]
I would like to see what contacts have visited on a specific date using the Tend API.  Is there an option for this?  So far I've only been able to loop through every contact, pull their visits, and see if they had one on the specific date I'm looking for.  At 700+ contacts and growing, it takes over 5 minutes to do this.  The export on the Tend.io site has a comma separated list inside a date column which would be perfect.  If that was something that could be added to the contact json info on the API, that would work also.

**Comment by purlem** [comment]
Hi Craig.  So something like `GET /visits?from={from date}&to={to date}` would be what you're looking for then?

**Comment by CraigH78** [comment]
Exactly.  For a single date, I would imagine you would use the same date in both from and to, or if I can include a timestamp that would work perfectly too. I could then say from = 2017-06-19 00:00:00 to = 2017-06-19 23:59:59.  

Thanks,
Craig

**Comment by CraigH78** [comment]
I'd need to be able to join the contact with the visit as well. Again, the end goal is to see who visited on a particular date.

**Comment by purlem** [comment]
@CraigH78 to/frame dates have been added: https://github.com/tendio/API/blob/master/README.md#visits

**Comment by CraigH78** [comment]
Works great, thank you!
